Tests/RosalindTests/RosalindAcceptanceTests.swift

```diff
@@ -6,44 +6,57 @@ import Rosalind
 import SnapshotTesting
 import Testing
 
-@Suite(.serialized) struct RosalindAcceptanceTests {
-    private let commandRunner = CommandRunner()
+struct RosalindAcceptanceTests {
     private let fileSystem = FileSystem()
     private let subject = Rosalind()
 
-    #if os(macOS)
-        @Test func apple_ios_app() async throws {
-            try await withFixtureInTemporaryDirectory("apple/ios_app") { temporaryDirectory, fixtureDirectory in
-                // Given
-                let xcodeprojPath = fixtureDirectory.appending(component: "App.xcodeproj")
-                let derivedDataDirectory = temporaryDirectory.appending(component: "derived-data")
-                try await commandRunner.run(
-                    arguments: [
```

Comment:
I got rid of this as for `.ipa`, we need the app signed. It felt this was adding unnecessary complexity for little benefits.